tests/regression/02-base/66-no-eval-on-write-multi.c

```diff
@@ -0,0 +1,36 @@
+//PARAM: --enable ana.int.interval --enable ana.int.enums --exp.privatization "write" -v
+
+#include<pthread.h>
+
+// Test case that shows how avoiding reading integral globals can reduce the number of solver evaluations.
+// Avoiding to evaluate integral globals when setting them reduced the number of necessary evaluations from 62 to 21 in this test case.
+
+pthread_mutex_t mutex = PTHREAD_MUTEX_INITIALIZER;
+int glob = 10;
+
+void* t_fun(void* ptr) {
+  pthread_mutex_lock(&mutex);
+  glob = 3;
+  glob = 4;
+  glob = 1;
+  pthread_mutex_unlock(&mutex);
+  return NULL;
+}
+
+void bar() {
+  glob = 2;
+}
+
+int main() {
+  pthread_t t;
+
+  pthread_create(&t, NULL, t_fun, NULL);
+  pthread_mutex_lock(&mutex);
+  bar();
+  pthread_mutex_unlock(&mutex);
+  pthread_join(t, NULL);
+  assert(glob >= 1);
+  assert(glob <= 2); //UNKNOWN
```

Comment:
Should this be `UNKNOWN!`?

Reply:
In the concrete semantics, the value of `glob` should either be `2`, from the write in `bar`, or `1`, from the last write in `glob`. AFAIK the `UNKNOWN!` is intended for something that is also unknown in the concrete, so that would not be the case. The abstraction used does not know about the `thread_join`, of course.
But it's probably less confusing if I remove the assert.

Reply:
Ahh, right, I didn't really look at the test.